satellite/metainfo/metainfo.go

```diff
@@ -1670,7 +1670,7 @@ func (endpoint *Endpoint) DownloadSegment(ctx context.Context, req *pb.SegmentDo
 		if segmentMeta != nil {
 			encryptedKeyNonce, err = storj.NonceFromBytes(segmentMeta.KeyNonce)
 			if err != nil {
-				return nil, status.Errorf(codes.Internal, err.Error())
+				return nil, status.Errorf(codes.Internal, "unable to get encryption key nonce from metadata: %v", err.Error())
```

Comment:
We are discussing in slack about to log the error for this specific internal errors https://storj.slack.com/archives/CAQV0AE2Z/p1564567179385100